### PR TITLE
Avatar and badge components

### PR DESCRIPTION
**Pull request by saurabhchirde**

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    font-size: 0.9em;
+    top: 0.3em;
+    right: 0.5em;
+    border-radius: 50%;
+}
+
+.btn-badge-cont button {
+    font-size: 1rem;
+    width: 100px;
+    padding: 6px 15px;
```
Comment (Ankur9669) [inline components/badge/badge.css]: If possible you can give padding in rems

File: components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    font-size: 0.8rem;
+    background-color: var(--badge-red);
+    position: absolute;
+    left: 0.9rem;
+}
+
+.btn-toggle-cont button {
+    font-size: 1rem;
+    width: 100px;
+    padding: 6px 15px;
```
Comment (Ankur9669) [inline components/badge/badge.css]: If possible you can give padding in rems

File: style.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+    overflow-y: scroll;
+}
+
+.nav::-webkit-scrollbar{
+    display: none;
+}
+
```
Comment (Ankur9669) [inline style.css:53]: If you wanna hide scroll bar in mozilla also add this code
/* Hide scrollbar for IE, Edge and Firefox */
.nav {
    -ms-overflow-style: none;  /* IE and Edge */
    scrollbar-width: none;  /* Firefox */
}

You can try opening you site in mozilla i think scrollbar would be there

File: index.html [hunk 4]
```diff
@@ -56,7 +86,7 @@ <h2 id="basic-alert" class="mg-2-top mg-point6-bot title-lg">
       </div>
 
       <iframe class="container-lg mg-1-tb" src="https://carbon.now.sh/embed/UhKee0hSJgkI1fGvO04l"
-        style="width: 992px; height: 250px; border:0; transform: scale(1)" sandbox="allow-scripts allow-same-origin">
+        style="width: 43rem; height: 250px; border:0; transform: scale(1)" sandbox="allow-scripts allow-same-origin">
```
Comment (Ankur9669) [inline index.html]: You can remove inline styling and create a css class for styling iframes

File: index.html [hunk 5]
```diff
@@ -173,112 +203,303 @@ <h2 class="title-lg mg-1-rt">Alert with actions</h2>
...
+        Avatars can be used to represent people or objects. It supports images or letters.
+      </p>
+
+      <!-- avatar as images -->
+      <h2 class="mg-1-tb">Image Avatar</h2>
+
+      <div class="cont-shadow component-container pd-1-all mg-1-bot">
+        <div class="avatar-round mg-1-tb mg-2-lr">
+          <div class="avatar avatar-lg"><img loading="lazy" src="components/images/avatar1.jpeg"
+              class="img-responsive img-round" alt="">
```
Comment (Ankur9669) [inline index.html]: 1. The indentation could be improved. 
2.  img tag does not have closing tag /.
3. alt tag is empty

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
Great work!!!, doing these changes would even make it better

**Comment by aiman-ansari** [comment]
Everything is good, naming is also good.
Thing need to change is avoid inline styling specially in iframe .
Make a separate css class to style your iframe.

**Comment by aiman-ansari** [comment]
Everything is good, naming is also good.
Thing need to change is avoid inline styling specially in iframe .
Make a separate css class to style your iframe.

**Comment by saurabhchirde** [comment]
Thank you @Ankur9669 and @aiman-ansari for taking out your time to review my code.
As per your provided comments and suggestion, I have made all changes. Let me know, if there are any or I can proceed to merge the branch.

**Review by saurabhchirde — COMMENTED** [review]
Updated with all the suggested changes.

**Pull request merged**